modules/pom.xml

```diff
@@ -23,6 +23,8 @@
         <aliases.xml.basedir>src/main/aliases</aliases.xml.basedir>
         <hibernate.search.version>${project.version}</hibernate.search.version>
         <hibernate.search.module.slot>${parsedVersion.majorVersion}.${parsedVersion.minorVersion}</hibernate.search.module.slot>
+        <infinispan.integration.slot>for-hibernatesearch-${hibernate.search.module.slot}</infinispan.integration.slot>
```

Comment:
In short, to break out of circular dependency, we should be releasing w/o assumptions on which Infinispan version is going to provide that module.
Encoding our version allows other projects to "provide something compatible to X", not least allows other projects to provide such modules for multiple versions (as needed and as tested).